modules/bullet/space_bullet.cpp

```diff
@@ -348,16 +348,46 @@ SpaceBullet::~SpaceBullet() {
 	destroy_world();
 }
 
+void SpaceBullet::add_to_pre_flush_queue(CollisionObjectBullet *p_co) {
+	if (p_co->into_flush == false) {
+		p_co->into_flush = true;
+		queue_pre_flush.push_back(p_co);
+	}
+}
+
+void SpaceBullet::add_to_flush_queue(CollisionObjectBullet *p_co) {
+	if (p_co->into_flush == false) {
+		p_co->into_flush = true;
+		queue_flush.push_back(p_co);
+	}
+}
+
```

Comment:
As mentioned above, should `into_flush` be used to indicated whether or not an item is already in either queue, or should there be separate indicators?

Reply:
More details above, but yes, same indicator is enough.